pkg/util/commons.go

```diff
@@ -159,6 +159,11 @@ func FormatPhone(phoneNumber string) string {
 	if phoneNumber == "" {
 		return ""
 	}
+
+	if !IsPhoneNumber(phoneNumber) {
```

Comment:
@chawocecil  Wouldn't this affect the logic where we use the `FormatPhone` function?